README.md

```diff
@@ -1,6 +1,6 @@
 # harp.gl [![Build Status](https://travis-ci.com/heremaps/harp.gl.svg?branch=master)](https://travis-ci.com/heremaps/harp.gl)
 
-`harp.gl` is an _experimental and work in progress_ open-source 3D map rendering engine.
+`harp.gl` is an _experimental and work in progress_ open-source 3D map rendering engine written in [TypeScript](https://github.com/microsoft/TypeScript).
```

Comment:
Maybe link the official page https://www.typescriptlang.org/?!

Reply:
I was thinking that the github page is better, but happy to change. @musculman do you prefer this link? If there is a majority I will change.

Reply:
Fine by me either way